actionmailer/lib/action_mailer/base.rb

```diff
@@ -664,11 +660,7 @@ def mailer_name
     # +nil+ in order to reset the value otherwise another field will be added
     # for the same header.
     def headers(args = nil)
-      if args
-        @_message.headers(args)
-      else
-        @_message
-      end
+      args.present? ? @_message.headers(args) : @_message
```

Comment:
Why is this checking for `present?` now? Do we want to handle `""` as same as `nil`? I don't think so.